kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -828,20 +828,23 @@ template<class TPointType> class Quadrilateral2D8
      */
     Matrix& InverseOfJacobian( Matrix& rResult, IndexType IntegrationPointIndex, IntegrationMethod ThisMethod ) const override
```

Comment:
I think that this function is well implemented in the base class. we could remove it from this specific file so that the base class implementation is used (to be checked)